templates/python-scrapy/src/items.py

```diff
@@ -12,3 +12,4 @@ class TitleItem(Item):
 
     url = Field()
     title = Field()
+    parsed_by = Field()
```

Comment:
Is this still needed with the simpler scraper?

Reply:
ofc not, good catch :)